src/item.cpp

```diff
@@ -1875,7 +1875,7 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
             info.emplace_back( "GUNMOD", _( "Handling modifier: " ), "",
                                iteminfo::show_plus, mod.handling );
         }
-        if( is_gun() && !type->mod->ammo_modifier.empty() && parts->test( iteminfo_parts::GUNMOD_AMMO ) ) {
+        if( !type->mod->ammo_modifier.empty() && parts->test( iteminfo_parts::GUNMOD_AMMO ) ) {
```

Comment:
Checking if ammo_modifier is empty is sufficient for all gunmods now that all gunmod items actually have `def.mod` instantiated, and makes sure that any gunmod which isn't itself a gun, but still modifies ammo, will work in the future (none exist currently)